apps/docs/content/components/collections/Listbox.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: ListBox
-description: A ListBox is a disclosure component that appears with a set of actions relevant to a specific control, interface area, data element or application view. Typically, this context is determined by the user’s current selection prior to invoking the menu. Listbox can be opened from components such as Selects or Buttons.
+description: A listbox displays actions relevant to the user’s current selection.
```

Comment:
I'm not sure I agree with using "listbox" here. Our component is called ListBox (same name as RAC), which implies that list and box are two separate words. By i do know that people often write listbox as one word.
so In our can, if you don't want to use the component name, I'd suggest putting "list box".

Reply:
https://react-spectrum.adobe.com/react-aria/ListBox.html I took inspiration from RAC actually.

Reply:
lol ok I still think it's weird, but whatever xD